Offbeat.GitWorkbench/RepositoryManagement/GitRepositoryView.xaml

```diff
@@ -39,11 +39,17 @@
 				<RowDefinition Height="*"></RowDefinition>
 			</Grid.RowDefinitions>
 
+			<StackPanel Grid.Row="0" Orientation="Vertical" 
+					HorizontalAlignment="Center" 
+					VerticalAlignment="Center"
+					Visibility="{Binding NotRepository, Converter={StaticResource BooleanToVisibility}}">
+				<TextBlock FontSize="20">The folder does not contain a valid Git repository</TextBlock>
```

Comment:
It would be more aesthetically pleasing if you ended the warning text with a full stop.